.env.sample

```diff
@@ -38,6 +38,7 @@ UKEF_TFM_API_REPORTS_KEY=
 SESSION_SECRET=
 
 # UTILISATION REPORT
+MAX_UTILISATION_REPORT_FILE_SIZE=1048576
```

Comment:
Could we update this to have the unit in the var name please?

Reply:
Yes, good idea 🧠